db/migrations/9.2.06.sql

```diff
@@ -0,0 +1,62 @@
+BEGIN;
+
+-- update secondary_keys_frozen field in identities table
+WITH event_data AS (
+  SELECT
+    event_id,
+	block_id,
+    event_arg_0 as did,
+    CASE
+      WHEN event_id = 'SecondaryKeysFrozen' THEN true
+      WHEN event_id = 'SecondaryKeysUnfrozen' THEN false
+      ELSE null
+    END as is_frozen,
+    created_at as updated_at
+  FROM events
+  WHERE event_id IN ('SecondaryKeysFrozen', 'SecondaryKeysUnfrozen')
+  ORDER BY created_at DESC
+)
+UPDATE identities
+SET
+  secondary_keys_frozen = event_data.is_frozen,
+  updated_block_id = event_data.block_id,
+  updated_at = event_data.updated_at
+FROM event_data
+WHERE identities.id = event_data.did;
+
+-- handle secondary key left identity 
+
+WITH event_data AS (
```

Comment:
I left some comments below, based on the discussion there we may require to remove the account entry as well or update something here